src/software/ai/hl/stp/tactic/receiver/receiver_fsm.h

```diff
@@ -74,8 +76,7 @@ struct ReceiverFSM
      * @param world The world to find a feasible shot on
      * @param assigned_robot The robot that will be performing the one-touch
      */
-    static std::optional<Shot> findFeasibleShot(const World& world,
-                                                const Robot& assigned_robot);
+    std::optional<Shot> findFeasibleShot(const World& world, const Robot& assigned_robot);
```

Comment:
update the docs because the constants that we refer to are removed

Reply:
you could update them in the `Passing` PR but I suspect you want that PR to be merged in

Reply:
Good call.

It actually turned out that receiver_tactic had a bunch of static functions that did not have a definition for a while, so I removed those as well.